.watch-app-format.json

```
@@ -0,0 +1,15 @@
+[
+  "trigger",
+  "./app",
+  {
+    "name": "format",
+    "expression": [
+      "anyof",
+      ["match", "**/*.js", "wholename"],
+      ["match", "**/*.json", "wholename"],
+      ["match", "yarn.lock", "wholename"]
+    ],
+    "command": ["/usr/bin/env", "bash", "-c", "yarn run format 1>../log/format.stdout.log 2>../log/format.stderr.log"],
```

`XO` prints a lot of spaces between the error message and the rule name, making the file hard to read with a limited width. This make it more readable. I can't find a way to set the eslint formatter in the `XO` config.
```suggestion
    "command": [
      "/usr/bin/env",
      "bash",
      "-c",
      "yarn run format | sed -E 's/\\s{2,}/\\t/g'  1>../log/format.stdout.log 2>../log/format.stderr.log"
    ],
```

---

the disadvantage to doing this is we break our row:column alignment

---

I see. I guess we don't really need to watch this file anyway. Not until the linter passes at least.